.github/workflows/build.yml

```diff
@@ -15,17 +15,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      # TODO: move to minimal Debian Docker container
-      # - name: Install dependencies
-      #   if: github.event_name != 'release'
-      #   run: |
-      #     sudo apt-get update
-      #     sudo apt-get install -y git-buildpackage
-      # - name: Patch changelog
-      #   if: github.event_name != 'release'
-      #   # '--ignore-branch' for testing on feature branch
-      #   run: |
-      #     gbp dch --ignore-branch --auto --snapshot --distribution=stable --force-distribution
+      - name: Patch changelog (in Debian Docker container)
+        if: github.event_name != 'release'
+        # '--ignore-branch' for testing on feature branch
+        run: |
+          docker create --name py-pitop-chnglg --volume ${{ github.workspace }}:${{ github.workspace }} --workdir ${{ github.workspace }}/ --tty pitop/gbp-dch-gen:latest sleep inf
+          docker start py-pitop-chnglg
+          docker exec py-pitop-chnglg git config --global user.name "GitHub Build Action"
+          docker exec py-pitop-chnglg git config --global user.email "https://github.com/pi-top/pi-top-Python-SDK"
```

Comment:
maybe better to use a valid email here like the deb-maintainers one

Reply:
This is only for preview builds - when we cut a release, we will use a proper email address. Would rather not include group email addresses in a public repo right now